Duplicati/Library/Backend/GoogleServices/GoogleCloudStorage.cs

```diff
@@ -106,22 +107,31 @@ private T HandleListExceptions<T>(Func<T> func)
                 if (wex.Response is HttpWebResponse response && response.StatusCode == HttpStatusCode.NotFound)
                     throw new FolderMissingException();
                 else
-                    throw;
+                {
+                    using (Stream exstream = wex.Response.GetResponseStream())

```

Comment:
I don't think you can guarantee there to be a response stream present? If the error is "Network disconnected" there will be no response stream and the user will get an exception here.

Since this logic is used multiple places, it might be a good idea to create a single `ParseErrorResponse(WebException wex)` helper.